src/instructlab/tax/diff.py

```diff
@@ -0,0 +1,90 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
I don't mind having: `taxonomy` instead of  `tax` directory. We don't have any  `tax` command either. So I'd like to keep the entire word.

Reply:
having `taxonomy` as the name breaks the gitignore. From what I can see (I may be wrong) there is no way to exclude `/taxonomy/` and not `/src/instructlab/taxonomy`. So I named it tax

Reply:
Would this solve your problem?

```
diff --git a/.gitignore b/.gitignore
index b650cec..1f492bd 100644
--- a/.gitignore
+++ b/.gitignore
@@ -3,6 +3,7 @@ models/
 !src/instructlab/train/lora_mlx/models
 # We recommend using taxonomy here, but not a submodule. Just a work dir.
 taxonomy/
+!src/instructlab/taxonomy/
 # We recommend putting generated files here, but PLEASE don't commit them.
 generated/
 # We recommend outputting training results on Linux here, but PLEASE don't commit them.
```

Then:

```
(venv) 👺  instructlab - [main] ❯❯❯ mkdir -p src/instructlab/taxonomy/ && touch src/instructlab/taxonomy/foo.py                  ✹
(venv) 👺  instructlab - [main] ❯❯❯ git status                                                                                 ✹ ✭
On branch main
Your branch is ahead of 'origin/main' by 453 commits.
  (use "git push" to publish your local commits)

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   .gitignore

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        src/instructlab/taxonomy/

no changes added to commit (use "git add" and/or "git commit -a")
```

Reply:
agree -- would prefer the `taxonomy` dir to match the command name (and the gitignore fix given by leseb)

Reply:
+1 let's keep `taxonomy`

Reply:
fixed!